spec/features/user/user_item_view_spec.rb

```diff
@@ -4,10 +4,12 @@
   before :each do
     @item = FactoryGirl.create(:item)
     user = FactoryGirl.create(:user)
-    visit login_path
-    fill_in("session_email", with: user.email)
-    fill_in("session_password", with: user.password)
-    click_button 'Sign in'
+    visit root_path
+    within("#signInModal") do
+      fill_in("session_email", with: user.email)
+      fill_in("session_password", with: user.password)
+      click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.